multiply.js

```diff
@@ -0,0 +1,4 @@
+function multiply(a,b) {
```

Comment:
- provide json documentaion for better understanding.

Reply:
resolved

Reply:
Look good to me